posthog/celery.py

```diff
@@ -95,6 +95,9 @@ def setup_periodic_tasks(sender: Celery, **kwargs):
     sender.add_periodic_task(120, clickhouse_mutation_count.s(), name="clickhouse table mutations count")
 
     sender.add_periodic_task(120, pg_table_cache_hit_rate.s(), name="PG table cache hit rate")
+    sender.add_periodic_task(
+        crontab(minute=0, hour="*"), pg_plugin_server_query_timing.s(), name="PG plugin server query timing"
```

Comment:
The metrics above use `60` or `120` seconds instead of `crontab`, why not follow that?

Reply:
These metrics change infrequently as they're aggregates over a large time period, so reducing load generated seems like a win.